src/main/java/io/github/minecraftcursedlegacy/example/ExampleMod.java

```diff
@@ -1,34 +1,23 @@
 package io.github.minecraftcursedlegacy.example;
-
-import java.io.IOException;
-
 import io.github.minecraftcursedlegacy.api.config.Configs;
 import io.github.minecraftcursedlegacy.api.registry.Id;
 import net.fabricmc.api.ModInitializer;
 import tk.valoeghese.zoesteriaconfig.api.container.WritableConfig;
 import tk.valoeghese.zoesteriaconfig.api.template.ConfigTemplate;
-
+import java.io.IOException;
 public class ExampleMod implements ModInitializer {
 	@Override
 	public void onInitialize() {
-		// This code runs as soon as Minecraft is in a mod-load-ready state.
```

Comment:
No justification for removing comments and changing formatt Ng

Reply:
Forgot I did that I'll add all the comments and such back.